js/src/components/experience-rating-banner/feedback-modal.js

```diff
@@ -27,12 +44,22 @@ const FeedbackModal = ( { onRequestClose } ) => {
 			) }
 			className="gla-experience-rating-banner__feedback-modal"
 			buttons={ [
-				<AppButton key="cancel" onClick={ onRequestClose } isTertiary>
+				<AppButton
+					key="cancel"
+					onClick={ () => {
```

Comment:
@mukeshpanchal27 Let's avoid anonymous functions and instead define separate functions for the `onClick` props.
```suggestion
					onClick={ handleOnCancelClick }
```